Sources/SwiftSyntaxMacroExpansion/MacroReplacement.swift

```diff
@@ -156,6 +178,45 @@ fileprivate class ParameterReplacementVisitor: SyntaxAnyVisitor {
     return .visitChildren
   }
 
+  override func visit(_ node: GenericArgumentClauseSyntax) -> SyntaxVisitorContinueKind {
+    return .visitChildren
+  }
+
+  override func visit(_ node: GenericArgumentListSyntax) -> SyntaxVisitorContinueKind {
+    return .visitChildren
+  }
+
+  override func visit(_ node: GenericArgumentSyntax) -> SyntaxVisitorContinueKind {
+    guard let baseName = node.argument.as(IdentifierTypeSyntax.self)?.name else {
+      // Handle error
```

Comment:
Is this a leftover todo?

Reply:
Wasn't sure what to do here, but actually best might be to just .skipChildren and call it a day